index.html

```diff
@@ -43,13 +43,23 @@ <h1 id="aemp-title">
         <details open>
           <summary>About the map:</summary>
           <div class="title-content-wrapper">
-            <h2>Emergency Tenant Protections</h2>
-            <p>
-              Where United States officials
-              <strong class="color-passed">have passed</strong> (or tenants are
-              <strong class="color-yet-to-pass">working to pass</strong>)
-              emergency tenant protections during the COVID-19 crisis.
-            </p>
+            <h2>Emergency Tenant Protections Scale</h2>
```

Comment:
Also @jimboweb please 
- leave the h2 title as it was before ("Emergency Tenant Protections")
- leave descriptive text as "Where officials have passed (or tenants are working to pass) emergency tenant protections during the COVID-19 crisis."

The legend HTML should go below the description and link to the form.